packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -673,6 +673,29 @@ export class PrismicRepositoryManager extends BaseManager {
 		}
 	}
 
+	async setDefaultMasterLocale(): Promise<void> {
+		const repositoryName = await this.project.getRepositoryName();
+
+		const localeServiceBaseUrl = new URL("locale/", API_ENDPOINTS.PrismicWroom);
```

Comment:
⚠️ Following this message from Michel, could you ensure you're using the right domain?

https://prismic-team.slack.com/archives/C039VR18ZDY/p1731667211291229

If we contact an internal service, as you see, there is a specific format instead of directly contacting wroom base url.

```
Format: [api.internal.prismic.io/](http://api.internal.prismic.io/)<service>
```

Reply:
url updated